content/rs/administering/designing-production/security/ldap-integration.md

```diff
@@ -18,8 +18,8 @@ LDAP authentication is not yet supported for Redis ACL Users.
 To configure LDAP authentication for RS web UI users on a running cluster:
 
 1. Configure `saslauthd` to use LDAP Authentication:
-    1. Edit the `/etc/default/saslauthd`, and change the `MECHANISMS` variable to `MECHANISMS="ldap"`.
-    1. Edit the `saslauthd.conf` configuration file in the installation directory (default: `/etc/opt/redislabs/saslauthd.conf`) and enter the values for these fields:
+    1. On each node, edit `/etc/default/saslauthd` to change the `MECHANISMS` variable to `MECHANISMS="ldap"`.
+    1. On one node, edit the `saslauthd.conf` configuration file in the installation directory (default: `/etc/opt/redislabs/saslauthd.conf`) and enter the values for these fields:
```

Comment:
Does it matter which node these changes are made on?

Reply:
Yes, there is a difference between some steps that you only perform on one node versus others that you perform on all nodes.